src/utils/hooks/useNotificationsInfiniteScroll.tsx

```diff
@@ -1,81 +1,30 @@
-import { useCallback, useContext, useEffect, useReducer, useRef, useState } from 'react'
+import { useEffect, useRef } from 'react'
 
-import W3iContext from '@/contexts/W3iContext/context'
-import { notificationsReducer } from '@/reducers/notifications'
+import { useNotifications } from '@web3inbox/react'
 
 const IntersectionObserverOptions = {
   root: null,
   rootMargin: '0px',
   threshold: 1.0
 }
 
-const NOTIFICATION_BATCH_SIZE = 12
+const NOTIFICATION_BATCH_SIZE = 6
```

Comment:
Context for the change? Also why 6 (would be good to add comment on why this number)